[WIP] listmintingについて相談

serisia opened the pull request:
xpc-wgにPRする前に・・・

- 名前
listmintingでいいかしら。

- 引数
今はlistunspentの頭にperiod（鋳造タブの期間指定と同じもの、分）をつけただけなんだけど
minconf、maxconfは正直鋳造っぽくないからminAge、maxAgeにしようかしら。
でもAge、今は日数なのよね。使いにくいかしら？

- 出力
何か過不足あるかしら。出力例をつけておくわ。
中身はregtestでやったものよ。

$ ./xpchain-cli listminting 2 2 3
```json
[
  {
    "txid": "5bd4238627c6fe822cc1508d196076a4ed4b2c9aa024f9e308f701b48181b51f",
    "vout": 0,
    "address": "xZhoYLVLEWpi1sbH7vZmRqfsncCHjzJh3C",
    "scriptPubKey": "2103530bdaffafb1e77c28399953064cc83a5a0ef45c707f4eb18447fb3299d25093ac",
    "amount": 85936.8320,
    "age": 0,
    "coinDay": 5474,
    "probability": 0.3899175225909158,
    "reward": {
      "minimum": 304,
      "maximum": 310
    }
  },
  {
    "txid": "6f68401a011850b356e9aab8f0d70cbf8d44c6be64ec13d143069d2e462069ad",
    "vout": 0,
    "address": "xZhoYLVLEWpi1sbH7vZmRqfsncCHjzJh3C",
    "scriptPubKey": "2103530bdaffafb1e77c28399953064cc83a5a0ef45c707f4eb18447fb3299d25093ac",
    "amount": 2749999.6660,
    "age": 0,
    "coinDay": 175153,
    "probability": 0.9999999550495713,
    "reward": {
      "minimum": 9744,
      "maximum": 9939
    }
  }
]
```

moochannel committed:
Added Dockerfile and compose sample file.

MaySoMusician committed:
Make addition of -minting argument in generating argument list for wallet

MaySoMusician committed:
Change category label for minting options from Stake to Minting

MaySoMusician committed:
Make -minting argument in MINTING category

MaySoMusician committed:
Make -minting description more clear

MaySoMusician committed:
Add -minting argument to dummy wallet

MaySoMusician committed:
Remove '=<n>' from description

serisia committed:
Modify Makefile for use kernelrecord in wallet rpc

serisia committed:
add rpc "listminting"

MaySoMusician commented:
Concept ACK
### ひとまずコマンド名について
既存の`list***`系RPCコマンドの命名規則は
 - *list received* transactions *by account* -> `listreceivedbyaccount`
 - *list received* transactions *by address* -> `listreceivedbyaddress`
 - *list unspent* transaction outputs -> `listunspent`
なので、今回は
 - *list* transaction outputs for *minting* -> `listminting`
ということで`listminting`で良いかと思われる

moochannel committed:
Expose RPC ports, but not publish them defaut

RPC ports can listen in Docker network. But they should not be connected
from outside.

moochannel committed:
Tweak publish port settings.

MaySoMusician committed:
Update translation definition file

MaySoMusician committed:
Update translation source file

yes-maxheart committed:
Merge pull request #55 from MaySoMusician/hotfix/minting-option

Wallet: Make description of -minting argument more clear

yes-maxheart committed:
Merge pull request #57 from moochannel/feature/dockerize

Scripts and tools: Add Dockerfile and compose sample file

yes-maxheart committed:
Merge pull request #58 from MaySoMusician/translate/shorter-minting-span

Translation: Change translation for shorter probability interval

nohtaehyun committed:
bugfit ko-KR.ts

yes-maxheart committed:
Merge pull request #56 from serisia/refactor/kernelrecord

Qt: refactoring KernelRecord

nohtaehyun committed:
KR-translation bugfix

yes-maxheart committed:
Merge pull request #59 from Falconno7/master

Translations: bugfix ko-KR.ts

serisia committed:
rename to endpoint from listminting to listminrings
